docs/demos/eye-tracking-with-webgazer.html

```diff
@@ -32,7 +32,7 @@
 
       var preload = {
         type: jsPsychPreload,
-        images: ['img/blue.png']
+        images: ['https://unpkg.com/jspsych@7.0.0/examples/img/blue.png']
```

Comment:
I don't think we need to load the media files from the CDN for the demo since we have an `/img` folder in `/docs/demos`. These can get pulled directly from jspsych.org/demos/img.

Reply:
Ah ok, I need to change this in a bunch of places then. Thanks!

Reply:
Oh, and the examples are [not included in the jspsych NPM package](https://github.com/jspsych/jsPsych/blob/2319a9a2118b1e919eb6af04b5d04baaac312940/packages/jspsych/package.json#L16-L20) (I think including them would unnecessarily bloat up the package).

Reply:
Ok thanks @bjoluc, that makes sense. (We were using the jsdelivr CDN before, which just hosted the whole github repo.)